src/popup.html

```diff
@@ -222,7 +222,7 @@ <h1 class="newsflash-headline i18n-content" data-i18n-message-id="popupDataColle
             <div class="newsflash-description">
               <h1 class="newsflash-headline i18n-content" data-i18n-message-id="popupPrivacyNoticeUpdateHeadline"></h1>
               <p class="newsflash-body i18n-content" data-i18n-message-id="popupPrivacyNoticeUpdateBody"></p>
-              <a class="newsflash-learn-more i18n-content" href="https://www.mozilla.org/privacy/firefox-relay/?utm_source=fx-relay-addon&utm_medium=popup&utm_content=privacy-notice-update-notification" data-i18n-message-id="popupPrivacyNoticeUpdateButtonLearnMore"></a>
+              <a class="newsflash-learn-more i18n-content" href="https://www.mozilla.org/privacy/firefox-relay/?utm_source=fx-relay-addon&utm_medium=popup&utm_content=privacy-notice-update-notification#relay" data-i18n-message-id="popupPrivacyNoticeUpdateButtonLearnMore"></a>
```

Comment:
I believe this is legacy code when we first changed our label storage policy default.

Reply:
String (`popupPrivacyNoticeUpdateBody`) referenced in this link:
> To improve the reliability of Firefox Relay in the event that an email can’t be delivered, we will temporarily keep the email on our servers until it has been delivered. We will never hold onto it for more than three days.